### PR TITLE
docs: sd quickfact docs [skip chromatic]

### PR DESCRIPTION
**Pull request by smfonseca**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
-import { waitUntil } from '@open-wc/testing-helpers';
-import cx from 'classix';
 
 const { argTypes, parameters } = storybookDefaults('sd-quickfact');
 const { overrideArgs } = storybookHelpers('sd-quickfact');
 const { generateTemplate } = storybookTemplate('sd-quickfact');
 
 /**
+ *
+ *  Used to display a quick fact item with an optional summary section.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:13]:
```suggestion
 *  Used to present information in a an expandable and more dynamic way.
```

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
+ *
+ *  **Related templates:**
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:15]: Please use the same quickfact sample as in figma

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 3]
```diff
@@ -54,411 +53,41 @@ export const Default = {
...
+    <sd-quickfact expandable="" summary="Lorem Ipsum">
+      <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div class="slot slot--border slot--text h-16">Default slot</div>
+    </sd-quickfact>
+  `
 };
 
 /**
- * sd-quickfacts are fully accessibile via keyboard.
+ * Use the `open` attribute to set the quickfact item to open by default.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:68]: Please use same samples as in FIGMA

Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:68]:
```suggestion
 * Use the `open` attribute to set the state of the quickfact.
```

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 4]
```diff
@@ -54,411 +53,41 @@ export const Default = {
...
+    <sd-quickfact expandable="" open summary="Lorem Ipsum">
+      <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div class="slot slot--border slot--text h-16">Default slot</div>
+    </sd-quickfact>
+  `
 };
 
 /**
- * This is a sample of how to group sd-quickfacts in a grid layout. Additional JavaScript is used to enable closing all other quickfacts when one is opened and to equalize the height of all summaries in a row. Open the "Show code" section to see the detailed implementation.
+ * Use the `summary` attribute to set the summary text or use the `summary` slot if you need to use custom markup.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:80]:
```suggestion
 * Use the `summary` to provide the text in the accordion quickfact.
Hint: The summary slot allows you to use custom html.
```

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 5]
```diff
@@ -54,411 +53,41 @@ export const Default = {
...
  */
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:81]: For the Summary sample please use the same as in FIGMA

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 6]
```diff
@@ -54,411 +53,41 @@ export const Default = {
...
-export const Sample = {
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:244]: Question: In code is the description included in the Summary?
In FIGMA we have 2 sections (please have a look)

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 7]
```diff
@@ -54,411 +53,41 @@ export const Default = {
...
-  name: 'Sample: Grouping',
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/quickfact/quickfact.stories.ts:245]: In figma we also have a responsive behaviour section.

**Review by Vahid1919 — APPROVED** [review]
Looks good from a technical point of view.

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 6 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:244]: What you describe in figma as "description" is done with the slot `summary`. 
If you want to create something similar to the description example you need to use **slot `summary`**, but if you want to create the summary example you can do it with the summary **attribute**. I hope this is clear, if not just give me call. FYI @coraliefeil

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 7 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:245]: As discussed on call, we do not document the responsive behaviour in storybook

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 2 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:15]: Updated

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 3 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts]: Updated

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 5 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:90]: Updated

**Review by Vahid1919 — APPROVED** [review]

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 6 above)
Comment (coraliefeil) [reply packages/components/src/components/quickfact/quickfact.stories.ts:244]: We inform about the deviation. All good

image

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 8]
```diff
@@ -54,411 +53,50 @@ export const Default = {
...
+        <div class="slot slot--border slot--text h-16">Default slot</div>
+      </sd-quickfact>
+    </div>
+  `
 };
 
 /**
- * Use the expand-icon and collapse-icon slots to change the expand and collapse icons, respectively.
- * To disable the animation, override the rotate property on the summary-icon part as shown below:
+ * Use the `summary` attribute to provide the text in the accordion quickfact.
```
Comment (coraliefeil) [inline packages/components/src/components/quickfact/quickfact.stories.ts:87]: * Use the `summary` attribute to provide the text in the quickfact.

File: packages/components/src/templates/quickfact.stories.ts [hunk 9]
```diff
@@ -0,0 +1,240 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Quickfact'
+};
+
+export const Default = {
+  render: () => html`
```
Comment (coraliefeil) [inline packages/components/src/templates/quickfact.stories.ts:10]: I refined the quickfact template.
Could you update code accordingly, pls?

The grouping should not wrap ... should stay in 1 row ... 

We can fill it with real content later.

File: packages/components/src/components/quickfact/quickfact.stories.ts (hunk 8 above)
Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:87]: Updated

File: packages/components/src/templates/quickfact.stories.ts (hunk 9 above)
Comment (smfonseca) [reply packages/components/src/templates/quickfact.stories.ts:10]: Updated

File: packages/components/src/components/quickfact/quickfact.stories.ts [hunk 10]
```diff
@@ -21,24 +23,21 @@ const { generateTemplate } = storybookTemplate('sd-quickfact');
 export default {
   title: 'Components/sd-quickfact',
```
Comment (coraliefeil) [inline packages/components/src/components/quickfact/quickfact.stories.ts:24]: Could you show the default in your intro, pls? 




Expandable is an optional feature.

Comment (coraliefeil) [reply packages/components/src/components/quickfact/quickfact.stories.ts:24]: When done -> Approved from design side. THX!

Comment (smfonseca) [reply packages/components/src/components/quickfact/quickfact.stories.ts:24]: Sorry, missed this one. 
Updated now.

**Pull request merged**